plugins/backstage-plugin-flux/src/hooks/useToggleSuspendResource.ts

```diff
@@ -1,9 +1,25 @@
-import { AlertApi, alertApiRef, useApi } from '@backstage/core-plugin-api';
+import {
+  AlertApi,
+  IdentityApi,
+  alertApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
 import { KubernetesApi, kubernetesApiRef } from '@backstage/plugin-kubernetes';
 import { CustomResourceMatcher } from '@backstage/plugin-kubernetes-common';
 import { useAsyncFn } from 'react-use';
 import { gvkFromKind } from '../objects';
 import { Deployment, Source } from '../components/helpers';
+import { useGetUserInfo } from './useGetUser';
+
+interface User extends IdentityApi {
+  result: {
```

Comment:
Ah! I guess the `result` is not really exposed by the FromLegacy stuff's API..

Can we use `getBackstageIdentity`  / `getProfileInfo` instead of accessing result directly?